src/frontend/sider/Fagsak/Behandling/context/BehandlingContext.tsx

```diff
@@ -271,23 +246,23 @@ export const BehandlingProvider = ({ children }: React.PropsWithChildren) => {
                 settIkkeKontrollerteSiderTilManglerKontroll,
                 søkersMålform,
                 trinnPåBehandling,
-                åpenBehandling,
+                behandling: behandling,
```

Comment:
Kan kanskje droppe denne

```suggestion
                behandling,
```